docs/builtin-plugins.md

```diff
@@ -256,13 +276,13 @@ For example. to transpile JSX, you can use this configuration.
 ]
 ```
 
-`limit2project` set to true, to use this plugin across an entire project (including other modules like npm)
+option `limit2project` can be set to true to use this plugin across an entire project (including other modules like npm)
```

Comment:
https://github.com/fuse-box/fuse-box/blob/97c3708f84241c13a4f6dd4ed73aca2b853d6655/src/plugins/BabelPlugin.ts#L26

is the opposite